config/wpcalypso.json

```diff
@@ -77,6 +77,7 @@
 		"me/trophies": false,
 		"network-connection": true,
 		"notifications2beta": true,
+		"notifications/link-to-reader": true,
```

Comment:
Should we also enable this for horizon? I tend to keep this in sync unless we have a reason not to.